extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -75,7 +75,11 @@ core::Property InvokeHTTP::ReadTimeout(
 core::Property InvokeHTTP::DateHeader(
     core::PropertyBuilder::createProperty("Include Date Header")->withDescription("Include an RFC-2616 Date header in the request.")->isRequired(false)->withDefaultValue<bool>(true)->build());
 
-core::Property InvokeHTTP::FollowRedirects("Follow Redirects", "Follow HTTP redirects issued by remote server.", "True");
+core::Property InvokeHTTP::FollowRedirects(
+  core::PropertyBuilder::createProperty("Follow Redirects")
+  ->withDescription("Follow HTTP redirects issued by remote server.")
+  ->withDefaultValue<bool>(true)
```

Comment:
According to Curl, this is false by default:
https://curl.se/libcurl/c/CURLOPT_FOLLOWLOCATION.html

So I think this is a breaking change, I would prefer to have false here by default.

Reply:
That's true, but I was checking the usage of this property in NiFi's version of [InvokeHTTP](https://nifi.apache.org/docs/nifi-docs/components/org.apache.nifi/nifi-standard-nar/1.13.2/org.apache.nifi.processors.standard.InvokeHTTP/index.html) and there the default value is True. Should we stil prefer the Curl's version for the default value?

Reply:
Hmm, good question. 
My reasoning was not based on Curl's default, but what we did before (which was based on Curl's default).
As minifi is still 0.xxx version, I tend to agree with you and introduce the breaking change to align with NiFi in this aspect.